File: assignment1/models/RawItem.java
package assignment1.models;

public class RawItem extends Item {
   private double tax;	
   public RawItem(double price) 
   {
		super(price);
		tax = (12.5 / 100) * price;
   }
   public Double fetchingTax() 
   {    
	    // instead of return the whole object, returned a variable of it
        return tax;
   }  
}

File: dart_assignment1/lib/component/raw_item.dart
import 'package:dart_assignment1/component/item_factory.dart';
import 'item.dart';

class RawItem extends Item {
  RawItem(String name, double price, int quantity, ItemType type)
      : super(name, price, quantity, type) {
    tax = ((12.5 / 100) * price);
  }
}
